### PR TITLE
[zh-cn]: update the description of the `new` operator

### PR DESCRIPTION
**Pull request by zhang-stone**
Update the description of the `new` operator

### CODE REVIEW

**Review by yin1999 — COMMENTED** [review]
Hey @zhang-stone. Could you update this part, as the translation does not match with the original [en-US content](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/new#description).

**Comment by zhang-stone** [comment]
Help me out. The markdownlint doesn't check out

File: files/zh-cn/web/javascript/reference/operators/new/index.md [hunk 1]
```diff
@@ -26,10 +26,13 @@ new constructor[([arguments])]
...
-2. 为步骤 1 新创建的对象添加属性 **`__proto__`**，将该属性链接至构造函数的原型对象；
-3. 将步骤 1 新创建的对象作为 **`this`** 的上下文；
-4. 如果该函数没有返回对象，则返回 **`this`**。
+1. 创建一个空的简单 JavaScript 对象（即 **`{}`**），为方便起见，我们称之为 `newInstance`；
+2. 如果构造函数的原型属性是一个对象，将 `newInstance` 的[[Prototype]]指向构造函数的原型属性，否则 `newInstance` 将保持为一个普通对象，其 [[Prototype]] 为 `Object.prototype`；
+
+  > **注意：**：因此，通过构造函数创建的所有实例都可以访问添加到构造函数原型属性中的属性/对象。
+
+3. 使用给定参数执行构造函数，并将 `newInstance` 绑定为 **`this`** 的上下文；
+4. 如果构造函数返回[非原始值](https://developer.mozilla.org/zh-CN/docs/Web/JavaScript/Data_structures)，则该返回值成为整个 `new` 表达式的结果。否则，返回 `newInstance`。（通常构造函数不返回值，但可以选择返回值，以覆盖正常的对象创建过程）
```
[the remark this answers is not in the document]
Comment (zhang-stone) [reply files/zh-cn/web/javascript/reference/operators/new/index.md:35]: @yin1999 Help me out, it looks fine, but the markdownlint check doesn't go through

[the remark this answers is not in the document]
Comment (yin1999) [reply files/zh-cn/web/javascript/reference/operators/new/index.md:35]: Don't worry. I'm going to take care of it

**Review by yin1999 — APPROVED** [review]
Thanks @zhang-stone :+1:

**Pull request merged**